docs/source/introduction/howdoesitwork.rst

```diff
@@ -42,9 +42,15 @@ Indemnification
 OpenKAT works with a system of indemnities for scanning, linked to intrusion levels. An organization gives an indemnification for a certain intensity of scan. For each intrusion level, the level is specified, in order to prevent unexpected problems for a production system.
 
 Intrusion levels or indemnities:
-:L1: 'do not touch'
+
+:L0: 'do not touch'
```

Comment:
I feel like we might need to explain this some more. Don't touch also applies to level 1, but level 0 never gathers information from this OOI. Whereas Level 1 does, but never directly by touching the system itself.

Reply:
Good point, will add this.